setup.cfg

```diff
@@ -23,6 +23,10 @@ classifier =
 packages =
     cellar
 
+[entry_points]
+console_scripts =
+    cellar = cellar.interfaces.main:run
```

Comment:
As a future note, the openstack way is s/interfaces/cmd for command line entry points.